lib/src/lib.rs

```diff
@@ -26,7 +27,7 @@ mod state_tree;
 
 /// Main interface to test `Actor`s with Kythera.
 pub struct Tester {
-    // Builtin actors root Cid used in the Machine
+    // Builtin test_actors root Cid used in the Machine
```

Comment:
are the builtin actors test actors?

Reply:
No ! Good one, must have been a wrong update on my side. Thanks for that :pray: